src/OSX/Avalonia.MonoMac/MonoMacPlatform.cs

```diff
@@ -87,8 +122,10 @@ namespace Avalonia
 {
     public static class MonoMacPlatformExtensions
     {
-        public static T UseMonoMac<T>(this T builder) where T : AppBuilderBase<T>, new()
+        public static T UseMonoMac<T>(this T builder, bool? useDeferredRendering = null) where T : AppBuilderBase<T>, new()
```

Comment:
Let's make the new parameter a `bool` with a default of `true` to match the usage in `Win32` and `Gtk3`.

Reply:
I think we should change Win32 and Gtk3 instead. The reasoning for that is that we probably want to be able to control some options with environment variables, but still provide a manual override from code. So `UseWhatever` should accept "I don't care" as a default value for `useDeferredRendering`.

Reply:
We can do that. Though I'd suggest using `AppContext` instead of environment variables when we get to that though.